server/src/resolvers/auditLogsResolver.ts

```diff
@@ -0,0 +1,73 @@
+import {AuditLogRepo} from "../repositories/AuditLogs/AuditLogRepository";
+
+const alr = new AuditLogRepo();
+
+const AuditLogResolvers = {
+    Query: {
+        auditLogs: async (_: any, args: any, context: any) => {
+            try {
```

Comment:
I would get rid of these try catch blocks, these stop the error messages from getting to the graphql playgound